### ISSUE TITLE
Configure TravisCI to build project changes

### ISSUE THREAD

**Issue by adam-becker** [adam-becker/psxact#15]
This is long overdue, as evidenced by #14 

Making this change should coincide with #13 so that the build results in library files instead of an executable.

**Comment by Thunder07** [comment]
We recently moved [Play! ](https:// github.com/jpd002/Play-) to cmake and setup CI using Travis and appvoyer, if you'd like I can  have a look and get this setup for you.

**Comment by adam-becker** [comment]
That would be fantastic! I've configured Travis before but I don't have very fond memories of it to say the least. Last I checked you had to use third party PPAs to get SDL2 from APT, and the only PPA I could find at the time was long since out of date.

I believe you'll need to be listed as a contributor to set that up though, correct?

**Comment by Thunder07** [comment]
> I believe you'll need to be listed as a contributor to set that up though, correct?

I don't think so, just activate appvoyer and travis for this repo, and it should be active from the moment I send you a PR.

> Last I checked you had to use third party PPAs to get SDL2 from APT, and the only PPA I could find at the time was long since out of date.

I haven't yet tried building on ubuntu, but according to [this](https://launchpad.net/ubuntu/trusty/+package/libsdl2-dev) it should be be available in default repo, the question is, if you need a specific (newer) version?

**Comment by adam-becker** [comment]
> I don't think so, just activate appvoyer and travis for this repo, and it should be active from the moment I send you a PR.

Okay, cool. I've enabled [Travis](https://travis-ci.org/adam-becker/psxact) and [Appveyor](https://ci.appveyor.com/project/adam-becker/psxact).

> I haven't yet tried building on ubuntu, but according to this it should be be available in default repo, the question is, if you need a specific (newer) version?

If I remember correctly the Travis team still has to whitelist certain packages, so even if they're in the APT repo, they might not be available to Travis builds. The last time I tried they were using `precise` but it seems they're upgrading to `trusty` so maybe it'll work, only one way to find out. :smile:
